cli/kubectl-hyperkube

```diff
@@ -140,24 +143,32 @@ def get(ctx, cluster):
              f'{param_string}'),
             headers=ctx.obj['headers']
         )
-        pprint(r.json())
+        if show_only:
```

Comment:
Should the print happen at all times? I feel like it should. why not change to the following?
```
pprint(r.json())
if not show_only:
    write_kubeconfig(r.json(), k8s_config)
```

Reply:
why should it happen all the time?

Reply:
It used to work that way? 🤷‍♂

Reply:
Ok there is some slight confusion about keeping backwards compatibility. This changes the behavior of past versions. A typical command would be the following. 
`kubectl-hyperkube -c ${HYPERKUBE_CONFIG} get --cluster ${CLUSTER_FQDN} > ${KUBECONFIG}`
If there is a print then it continues to work. But there may be unexpected behavior because of the merge. I feel like old behavior should be the default and the merge should be an option.

Reply:
ok. changing the default to display to console and merge write.

Reply:
done. merge is not default behavior.

Reply:
👍 👏